cdk/lib/prism-ec2-app.ts

```diff
@@ -0,0 +1,80 @@
+import type { CfnAutoScalingGroup } from "@aws-cdk/aws-autoscaling";
+import { BlockDeviceVolume, EbsDeviceVolumeType } from "@aws-cdk/aws-autoscaling";
+import { Peer } from "@aws-cdk/aws-ec2";
+import type { App } from "@aws-cdk/core";
+import { Duration } from "@aws-cdk/core";
+import { AccessScope, GuPlayApp } from "@guardian/cdk";
+import type { AppIdentity } from "@guardian/cdk/lib/constructs/core/identity";
+import type { GuStackProps } from "@guardian/cdk/lib/constructs/core/stack";
+import { GuStack } from "@guardian/cdk/lib/constructs/core/stack";
+import {
+  GuAllowPolicy,
+  GuAssumeRolePolicy,
+  GuDynamoDBReadPolicy,
+  GuGetS3ObjectsPolicy,
+} from "@guardian/cdk/lib/constructs/iam";
+
+export class PrismEc2App extends GuStack {
+  private static app: AppIdentity = {
+    app: "prism",
+  };
+
+  constructor(scope: App, id: string, props: GuStackProps) {
+    super(scope, id, props);
+
+    const pattern = new GuPlayApp(this, {
+      ...PrismEc2App.app,
+      userData: {
+        distributable: {
+          fileName: "prism.deb",
+          executionStatement: `dpkg -i /${PrismEc2App.app.app}/prism.deb`,
+        },
+      },
+      certificateProps: {
```

Comment:
We will now create our own cert within the template, instead of passing in the arn via a parameter. We should be able to remove an unused cert when the migration process is complete.

Reply:
Unused certificates have now been removed manually.